docs/layouts/index.html

```diff
@@ -92,6 +90,16 @@ <h2 class="mt-6 text-xl mx-auto text-center font-bold text-gray-400 leading-6 tr
     </section>
   </div>
 </div>
+<div>
+  <div class="mt-6 grid grid-cols-1 gap-4 w-full">
+    <h2 class="mt-6 text-xl mx-auto text-center font-bold text-gray-400 leading-6 tracking-tighter bg-clip-text text-transparent bg-gradient-to-b from-gray-900 to-gray-600 dark:from-gray-100 dark:to-gray-400" style="max-width: 800px">
+     We are a <a href="https://www.cncf.io/">Cloud Native Computing Foundation</a> Sandbox Project
+     <br>
+     <br>
+     <center> <img href="https://www.cncf.io/" src="/images/cncf-color.svg" alt="We are a Cloud Native Computing Foundation Sandbox Project" width="350" height="350"> </center>
```

Comment:
Another problem is in dark mode this gets washed out, there is a black and white CNCF logo but I don't know how to switch it over when in dark mode

Reply:
I will fix this.